Cargo.lock

```diff
@@ -3728,9 +3794,9 @@ dependencies = [
 
 [[package]]
 name = "librocksdb-sys"
-version = "6.11.4"
+version = "6.17.3"
```

Comment:
you seem to have bumped all of the dependencies with `cargo update` instead of `cargo update -p pallet-beefy`, which makes the review harder

Reply:
Unfortunately, just bumping BEEFY won't compile due to some libp2p issues in authority discovery. So I assume, just bumping BEEFY will lead to inconsistencies. Bump Substrate for BEEFY right before the PR, so diffs should be relatively contained.

Reply:
This is probably because https://github.com/paritytech/grandpa-bridge-gadget/pull/154 was updated with `cargo update` rather than `cargo update -p sp-io`. If that was the case then let's revert that PR and then redo this one. We shouldn't yolo update all dependencies as that may have implications to the stability of the software and needs to be properly tested.

Reply:
I don't think `Cargo.lock` in BEEFY would affect `Cargo.lock` in Polkadot, because that's not the case for Substrate.